bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -7160,52 +7176,50 @@ components:
       required:
         - isShipperOwned
 
-    EmptyContainerPositioning:
+    ContainerPositioning:
       type: object
-      title: Empty Container Positioning
+      title: Container Positioning
       description: |
-        An object to capture the `Empty Container Positioning Location` together with an optional `Empty Container Positioning Date and Time`.
+        An object to capture the `Location` together with an optional `Date and Time`.
       properties:
-        emptyContainerPositioningDateTime:
+        dateTime:
           type: string
           format: date-time
           description: |
-            The date and time requested by the shipper for the positioning of the empty container(s) at the `Empty Container Positioning Location` (ECP), if provided, or the `Place of Receipt` (PRE) if ECP location is not provided.
-            
-            **Conditional:** Only applicable to carrier haulage service at origin (`Receipt type at origin = SD`).
+            The date and time requested by the shipper for the positioning of the container(s) at the Container positioning location (`CPO`), if provided, or the Place of Receipt (`PRE`) if `CPO` location is not provided.
           example: '2024-07-05T09:03:00-02:00'
-        emptyContainerPositioningLocation:
-          $ref: '#/components/schemas/EmptyContainerPositioningLocation'
+        location:
+          $ref: '#/components/schemas/ContainerPositioningLocation'
       required:
-        - emptyContainerPositioningLocation
+        - location
 
-    EmptyContainerPositioningEstimated:
+    ContainerPositioningEstimated:
       type: object
-      title: Empty Container Positioning
+      title: Container Positioning
```

Comment:
The title - Container Positioning Estimated

Reply:
Fixed